test/oqs_test_kems.c

```diff
@@ -27,12 +27,13 @@ static int test_oqs_kems(const char *kemalg_name) {
         return 1;
     }
     // test with built-in digest only if default provider is active:
-    // TBD revisit when hybrids are activated: They always need default
-    // provider
+    // limit testing to oqsprovider as other implementations may support
+    // different key formats than what is defined by NIST
```

Comment:
Or different sets of parameters, regardless of whether the key formats are the same.

Reply:
Yup -- as was the case here. Using https://github.com/open-quantum-safe/oqs-provider/issues/612 to track next steps. OK to mark resolved?

Reply:
@vdukhovni Please mark this resolved if you're OK with the above.

FYI, in `oqsprovider` only the people raising issues are meant to mark them resolved. Committers try to do that only when they're convinced the originator has been run over by the proverbial bus (e.g., when they drove it :-)

Reply:
Closing as per direct discussion with @vdukhovni